simulator/src/themer/themer.js

```
@@ -49,7 +49,14 @@ function updateThemeForStyle(themeName) {
     Object.keys(selectedTheme).forEach((property, i) => {
         html.style.setProperty(property, selectedTheme[property]);
     });
-    colors = getCanvasColors();
+     const newColors = getCanvasColors();
```

ESLint

[error] Expected indentation of 4 spaces but found 5.

(indent)